spec/mongoid/validatable/uniqueness_spec.rb

```diff
@@ -2472,4 +2472,32 @@ class SpanishActor < EuropeanActor
       }.to raise_error(Mongo::Error::OperationFailure)
     end
   end
+
+  describe "i18n" do
+
+    before do
+      Dictionary.validates_uniqueness_of :name
+    end
+
+    after do
+      Dictionary.reset_callbacks(:validate)
+    end
+
+    around do |example|
+      I18n.with_locale(:fr) { example.run }
+    end
+
+    before do
+      # Translation key location is as per rails-i18n gem.
+      # See: https://github.com/svenfuchs/rails-i18n/blob/master/rails/locale/en.yml
+      I18n.backend.store_translations(:fr, { errors: { messages: { taken: 'est déjà utilisé(e)' } } })
+    end
+
+    it "uses the correct language translation" do
```

Comment:
This is missing the `context` describing the context, i.e. "when the taken validation message is translated"

Reply:
Done.